examples/twittermap/web/public/javascripts/mapresultcache/services.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Module to cache map result data. Common services module communicates with
+ * middleware and updates resultcache module.
+ */
+'use strict';
+angular.module('cloudberry.mapresultcache', ['cloudberry.common'])
+  .service('MapResultCache', function () {
+
+    // The key-value store that stores mapResults
+    var store = new HashMap();
+    // To check if keyword in query changed
+    var currentKeywords = [""];
+    // To check if time range in query changed
+    var date = new Date();
+    date.setDate(date.getDate() - 1);
```

Comment:
why `-1`? is it the last day?

Reply:
Taewoo had suggested that we add the default as impossible dates where start date is current timestamp and end date is the previous day's timestamp.